ashleyo.txt

```diff
@@ -0,0 +1 @@
+This is a file created by Ashley O'Mahony.
```

Comment:
Oops, my bad - I didn't notice this file should have been in the `names` directory https://github.com/Juanlu001/ie-mbd-advanced-python/tree/master/names Volunteer to fix it?

Reply:
I tried, but I see while pulling the master that it's already done 😀

Reply:
True :slightly_smiling_face: